OpenRA.Mods.RA/Traits/Harvester.cs

```diff
@@ -137,11 +137,8 @@ Actor ClosestProc(Actor self, Actor ignore)
 						return occupancy * 12;
 					}));
 
-			// Reverse the found-path to find the refinery location instead of our location:
-			path.Reverse();
-
```

Comment:
Thanks for the clarification. I thought `path` was re-used later and the harvester would otherwise not find it's way home.